docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -10,8 +10,34 @@ document are written by the developer most familiar with the change made.
 The contribution should be included as part of the Iris Pull Request that
 introduces the change.
 
-The ``latest.rst`` and the past release notes are kept in 
-``docs/iris/src/whatsnew/``.
+The ``latest.rst`` and the past release notes are kept in
+``docs/iris/src/whatsnew/``. If you are writing the first contribution after
+an Iris release: **create the new** ``latest.rst`` by copying the content from
+``latest.rst.template`` in the same directory.
+
+Since the `Contribution categories`_ include Internal changes, **all** Iris
+Pull Requests should be accompanied by a "What's New" contribution.
+
+
+Git Conflicts
+=============
+
+If changes to ``latest.rst`` are being suggested in several simultaneous
+Iris Pull Requests, Git will likely encounter merge conflicts. In you think
+this situation is likely (large PR, high repo activity etc.): wait until your
+code is ready for review, then create a **new pull request** specifically for
+the "What's New" entry, referencing the main pull request and titled
+(e.g. for PR#9999):
+
```

Comment:
Sometimes the review can take a while, e.g. because no reviewers are available or because the change turns out to be more complicated than the developer realised (the latter happens to me a lot!)  I wonder if it would be better to leave the whatsnew PR until the main one is almost done.

Reply:
I agree this is a problem. But it would be ideal if the reviewer would still be able to insist on a "What's New" before merging. Do you think it would work for the timeline to look something like: "OK I'm now happy we've sorted the code changes, please can you create a "What's New" PR and we'll be done."?

Reply:
That seems reasonable to me.  Having said that, if the main PR is almost ready to merge, it might be simpler to rebase and add the whatsnew to the existing branch.

Rebase would also ensure that Travis is looking at the right thing.  I did once manage to [break master with a failure that didn’t show up in my PR](https://github.com/SciTools/iris/pull/2960#issuecomment-385633071). 😬

Reply:
>Rebase would also ensure that Travis is looking at the right thing. I did once manage to break master with a failure that didn’t show up in my PR. 😬

As I understand, Travis simulates a merge with the default branch, then runs the tests. So rather than rebasing I **think** the problem you reference would have been caught by re-spinning Travis just before merging the PR.

Reply:
8158666c0a8432af262e9141c7167975501a61b4